rosetta/docs/DEVELOPMENT.md

```diff
@@ -0,0 +1,90 @@
+# Development
+This repo builds upon [t5x](https://github.com/google-research/t5x) and patches it with features
+that are necessary to train the models defined in this repo.
+
+If a PR to T5X upstream takes time to be merged, this is fine as this project follows the
```

Comment:
This reads like an internal guideline. Since this repo is public, shall we reword it a bit to give people from outside some context?

Reply:
I changed it to this after @nouiz 's feedback. Perhaps I can propose this wording:
```text
This project follows the [distribution model](https://opensource.com/article/18/7/forks-vs-distributions). This means
we cherry-pick the necessary changes onto the tip of upstream t5x. These changes are typically available as PRs
against upstream [t5x](https://github.com/google-research/t5x).
```